src/camodocal/calib/HandEyeCalibration.h

```diff
@@ -3,7 +3,7 @@
 
 #include <Eigen/Eigen>
 #include <Eigen/StdVector>
-
+#include "ceres/ceres.h"
```

Comment:
`#include <ceres/ceres.h>` since it is an external library, at this point quotes have been used for internal headers only..